spec/dummy/db/test_track_schema.yml

```diff
@@ -1,9 +1,23 @@
 ---
 splits:
   buy_one_get_one_promotion_enabled:
-    'false': 50
-    'true': 50
+    weights:
+      'false': 50
+      'true': 50
+    hypothesis: "user will buy more hats with promotion"
+    location: "hat store"
+    platform: "desktop"
+    owner: "hat makers"
+    assignment_criteria: "user has bought one hat"
+    description: "promotion test to see if users will buy more hats"
```

Comment:
is this actually getting dumped into the schema.yml now? i don't think that it is and i don't think that it should be.

the schema.yml is only meant to track the critical details of splits, their variants, and the associated weighting. we don't want it to include any of the optional split detail data that is inputted through the admin ui.